### PR TITLE
Update camera example to support face detection and recognition

### PR DESCRIPTION
**Pull request by me-no-dev**
Fixes: #6508

### CODE REVIEW

**Comment by HaydenHu** [comment]
The enroll face button cannot register faces

**Comment by me-no-dev** [comment]
recognition does not work at all on chips other than ESP32-S3. Is that what you are running?

**Comment by HaydenHu** [comment]
image
I'm using this development board, which uses esp32, not esp32s3

**Comment by me-no-dev** [comment]
The updated AI library for image recognition does work on ESP32, but it would take 20 seconds to run a single 320x240 frame (compared to a few hundred milliseconds on ESP32-S3), so I have disabled it on ESP32 and ESP32-S2 to save flash space and memory. If you really want face recognition, maybe go back to 1.0.6.

File: libraries/ESP32/examples/Camera/CameraWebServer/partitions.csv [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Name,   Type,  SubType, Offset,   Size,    Flags
+nvs,      data,  nvs,     0x9000,   0x5000,
+otadata,  data,  ota,     0xe000,   0x2000,
+app0,     app,   ota_0,   0x10000,  0x3d0000,
+fr,       data,        ,  0x3e0000, 0x20000,
```
Comment (knopkakhua) [inline libraries/ESP32/examples/Camera/CameraWebServer/partitions.csv:5]: coredump, data, coredump,0x3F0000,0x10000,

**Pull request merged**